web/webapp_content/app/report/remove/remove-report.controller.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('org.perfrepo.report')
+        .controller('RemoveReportController', RemoveReportController);
+
+    function RemoveReportController(_report, reportService, $uibModalInstance) {
```

Comment:
Just out of my curiosity, is this some sort of convention in Angular, to start is with underscore? Why is it used only here?